src/v2/Apps/Artist/Routes/AuctionResults/Components/MarketStats.tsx

```diff
@@ -224,7 +224,17 @@ export const MarketStatsFragmentContainer = createFragmentContainer(
 export const MarketStatsQueryRenderer: React.FC<{
   artistInternalID: string
   environment: RelayModernEnvironment
-}> = ({ artistInternalID, environment }) => {
+  onRendered?: () => void
+}> = ({ artistInternalID, environment, onRendered }) => {
+  const [hasRendered, setHasRendered] = useState(false)
+
+  const onRender = () => {
+    if (hasRendered) return
+
+    setImmediate(() => setHasRendered(true))
```

Comment:
Using `setImmediate` to avoid the "Cannot update state while rendering a different component" warning.

Reply:
Nice, I think this approach might fix another area where this warning throws 👍 thanks for the comment here about that.